src/core/audio/AudioPulseAudio.cpp

```diff
@@ -265,35 +306,59 @@ void AudioPulseAudio::streamWriteCallback( pa_stream *s, size_t length )
 {
 	const fpp_t fpp = mixer()->framesPerPeriod();
 	surroundSampleFrame * temp = new surroundSampleFrame[fpp];
```

Comment:
We need to memset it to zero, or just make sure getNextBuffer always fill the buffer with zeros if there is no data.